source/common/router/vhds.h

```diff
@@ -39,9 +39,11 @@ struct VhdsStats {
 class VhdsSubscription : Envoy::Config::SubscriptionBase<envoy::config::route::v3::VirtualHost>,
                          Logger::Loggable<Logger::Id::router> {
 public:
-  VhdsSubscription(RouteConfigUpdatePtr& config_update_info,
-                   Server::Configuration::ServerFactoryContext& factory_context,
-                   const std::string& stat_prefix, Rds::RouteConfigProvider* route_config_provider);
+  static absl::StatusOr<std::unique_ptr<VhdsSubscription>>
```

Comment:
nit:
s/std::unique_ptr<VhdsSubscription>/VhdsSubscriptionPtr/

Reply:
VhdsSubscriptionPtr isn't declared untl after the class - would have to predeclare the class and move up the using=